sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/src/Bindings/ConnectResponse.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.Azure.WebJobs.Extensions.WebPubSub
+{
+    [JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy))]
+    public class ConnectResponse : ServiceResponse
```

Comment:
Does this need to be public?

Reply:
Yes. Customer need to return different response in different scenarios. See [sample usage](https://github.com/JialinXin/wps-function-poc/blob/main/samples/simplechat/csharp/Functions.cs#L23-L35).